Gemfile.lock

```diff
@@ -182,10 +183,9 @@ GEM
       mime-types
       mimemagic (~> 0.3.0)
       terrapin (~> 0.6.0)
-    listen (3.1.5)
+    listen (3.0.8)
```

Comment:
Seems strange that this would downgrade the version

Reply:
yes it is however listen is a development dependency that plays no role in the production environment.